teslajsonpy/sentry_mode.py

```diff
@@ -31,6 +31,7 @@ def __init__(self, data, controller):
         """
         super().__init__(data, controller)
         self.__manual_update_time = 0
+        self.__sentry_mode_available = False
```

Comment:
Isn't this unnecessary since it's in vehicle?

Reply:
Correct, it is in VehicleDevice as `sentry_mode_available`. `__sentry_mode_available` can be removed.